app/mmConfig.js

```diff
@@ -1,23 +1,18 @@
+require('dotenv').config()
 module.exports = {
-  //
   // Tokenlon server address
   EXCHANGE_URL: process.env.EXCHANGE_URL,
   PROVIDER_URL: process.env.PROVIDER_URL,
 
-  //
   // Wallet
   WALLET_ADDRESS: process.env.WALLET_ADDRESS,
-  USE_KEYSTORE: true,
+  USE_KEYSTORE: !process.env.USE_KEYSTORE || JSON.parse(process.env.USE_KEYSTORE),
   WALLET_KEYSTORE: {},
```

Comment:
here is the public version for market maker, this option MUST on by default.

Reply:
removed